Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -34,6 +34,9 @@ public protocol Environmenting: AnyObject {
 
     /// Returns the path to the directory where the async queue events are persisted.
     var queueDirectory: AbsolutePath { get }
+    
+    /// Returns true if the environment is ARM.
+    var isArm: Bool { get }
```

Comment:
What about using an "Architecture" enum here instead of a boolean? I guess we shouldn't expect more processor architectures, but at least in term of readability I think that the enum reads better than a bool.